articles/manual/statements.md

```diff
@@ -658,7 +712,41 @@ from qubits that are in use but that are not accessed during the body of the `bo
 If there aren't enough such qubits, then it will allocate new qubits
 to complete the request.
 
-## Expression Evaluation Statements
+
+### Intrinsic Operations
+Once allocated, a qubit can then be passed to functions and operations.
+In some sense, this is all that a Q# program can do with a qubit, as the actions that can be taken are all defined as operations.
+More information of these operations can be found in [Intrinsic Operations and Functions](xref:microsoft.quantum.libraries.standard.prelude) and [Operations and Functions](xref:microsoft.quantum.manual.operations-functions).
+
+Here we present an example of operation implementation:
+First, the single-qubit Pauli operators $X$, $Y$, and $Z$ are represented in Q# by the intrinsic operations `X`, `Y`, and `Z`, each of which has type `(Qubit => Unit is Adj + Ctl)`.
```

Comment:
It might be good to link to the API references for `X`, `Y`, and `Z` here.

Reply:
Ok. Done in 6d27732ae2a8bf026a01f135d6fd4e7891328e2c